hibernate-core/src/main/java/org/hibernate/persister/entity/UniqueKeyLoadable.java

```diff
@@ -16,7 +17,11 @@
 	 * Load an instance of the persistent class, by a unique key other
 	 * than the primary key.
 	 */
-	Object loadByUniqueKey(String propertyName, Object uniqueKey, SharedSessionContractImplementor session);
+	Object loadByUniqueKey(String propertyName, Object uniqueKey,
```

Comment:
Format thing... if you are going to "chop down" arguments, can you please chop them all down?  

E.g.

```
	Object loadByUniqueKey(
		String propertyName,
		Object uniqueKey,
		SharedSessionContractImplementor session);

	Object loadByUniqueKey(
		String propertyName,
		Object uniqueKey, 
		LockOptions lockOptions,
		SharedSessionContractImplementor session);
```

Reply:
Fixed.